### DISCUSSION TITLE
求助，这咋办

### DISCUSSION THREAD

**Discussion by dong200497** [share121/inter-knot#4546] [委托]
![image](https://github.com/user-attachments/assets/e34ab419-5521-46e1-a7af-021e1d9b568e)

**Comment by Dave-12138** [comment]
浏览器被跨域拦了？现在网页绳网没反向代理，只能[装这个浏览器插件](https://microsoftedge.microsoft.com/addons/detail/allow-cors-accesscontro/bhjepjpgngghppolkjdhckmnfphffdag?hl=zh-CN&gl=CN)凑合一下<img src="https://bbs-static.miyoushe.com/static/2024/09/04/a9fbeee1e21cf1abcf7bf2e610ee91f5_8437422338490524317.png" alt="玛拉妮-气氛组" width="75" >

**Reply by Dave-12138** [reply]
如果你是pc客户端这样，那大概是因为客户端连不上github，需要调教一下你的代理<img src="https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png" alt="流萤 也挺好" width="75" >

**Reply by dong200497** [reply]
下上插件就好了，非常感谢
